doc/whats_new/v1.1.rst

```diff
@@ -117,6 +117,14 @@ Changelog
   backward compatibility, but this alias will be removed in 1.3.
   :pr:`21177` by :user:`Julien Jerphanion <jjerphan>`.
 
+:mod:`sklearn.manifold`
+.......................
+
+- |Enhancement| :func:`manifold.spectral_embedding` and
+   :class:`manifold.SpectralEmbedding` supports `np.float32` dtype and will
+   preserve this dtype.
+  :pr:`21534` by :user:`Andrew Knyazev <lobpcg>`.
```

Comment:
The CI fails because of this one (that I probably introduced in my suggestion :))

```suggestion
- |Enhancement| :func:`manifold.spectral_embedding` and
  :class:`manifold.SpectralEmbedding` supports `np.float32` dtype and will
  preserve this dtype.
  :pr:`21534` by :user:`Andrew Knyazev <lobpcg>`.
```

Reply:
yes, just fixed it already